ui/logviewer/App.jsx

```diff
@@ -214,6 +223,23 @@ class App extends React.PureComponent {
     }
   };
 
+  logFormatter = (log) => {
+    if (/INFO - TEST-(OK)|(PASS)/.test(log))
+      return <span style={{ color: 'green' }}>{log}</span>;
+    if (/INFO - TEST-START/.test(log))
+      return <span style={{ color: 'blue' }}>{log}</span>;
+    if (/INFO - TEST-/.test(log))
+      return <span style={{ color: 'yellow' }}>{log}</span>;
```

Comment:
Yellow and white are not readable color for text, so you'll need to choose other colors: http://localhost:5000/logviewer.html#/jobs?job_id=306517762&repo=autoland&lineNumber=2182

For info perhaps it can have a black text with a light gray background color so it looks like a highlight.